nixpkgs/default.nix

```diff
@@ -0,0 +1,19 @@
+# How to update this?
+#
+# 1. Find the new desired commit hash on github.com/nixos/nixpkgs.
```

Comment:
I don't understand which sha1 to use where and when.

Reply:
I updated the doc a bit, but I need further help to know where your issue is.